gnomad/utils/constraint.py

```diff
@@ -23,12 +30,13 @@ def annotate_with_mu(
 
     .. note::
 
-        `ht` is expected to include annotations that `mutation_ht` is keyed by, but these annotations don't need to be
-        the keys of `ht`.
+        `ht` is expected to include annotations that `mutation_ht` is keyed by, but
```

Comment:
I wrapped the docstrings for all the functions in this file. Please ignore the functions that are out of the scope of this PR. Their contents weren't changed at all!

Reply:
thanks for the heads up! as a quick poinrt of feedback: these docstring changes made the PR a lot harder to read through

Reply:
So sorry for the inconvenience! Maybe I should separate it to another PR next time.